packages/opentelemetry-sdk-node/test/sdk.test.ts

```diff
@@ -128,4 +169,233 @@ describe('Node SDK', () => {
       assert.ok(metrics.getMeterProvider() instanceof MeterProvider);
     });
   });
+
+  describe('detectResources', async () => {
```

Comment:
All of the old `detect-resources.test.ts` tests required the actual detectors, so I moved these tests from `@opentelemetry/resources` tests to `sdk.test.ts`. This move affects codecov, so eventually new tests with test-stub detectors should be added to the resources package

Reply:
I agree. We should add a couple stub detectors like one that fails and one that succeeds.